internal/descriptor/cache/cache.go

```diff
@@ -0,0 +1,34 @@
+package cache
+
+import (
+	"sync"
+
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+)
+
+type DescriptorCache struct {
+	cache sync.Map
```

Comment:
What about having this cache as a ttl cache?

Reply:
Good idea, but let's change that implementation after the refactoring!
After these changes changing the internals of the cache is much nicer!